moped-editor/src/views/projects/projectView/ProjectTeamRoleMultiselect.js

```diff
@@ -71,15 +59,29 @@ const ProjectTeamRoleMultiselect = ({
           },
         }}
       >
-        {Object.keys(roles).map(roleId => (
-          <MenuItem key={roleId} value={Number.parseInt(roleId)}>
-            <Checkbox
-              checked={userRoles.includes(Number.parseInt(roleId))}
-              color={"primary"}
-            />
-            <ListItemText primary={roles[roleId]} />
-          </MenuItem>
-        ))}
+        {roles.map(
+          ({
+            project_role_id,
+            project_role_name,
+            project_role_description,
+          }) => {
+            const isChecked = value.includes(project_role_id);
+            return (
+              // ListItemClasses
+              <MenuItem key={project_role_id} value={project_role_id}>
+                <Checkbox checked={isChecked} color={"primary"} />
+                <ListItemText
+                  primary={<span>{project_role_name}</span>}
+                  secondary={
+                    <span style={{ whiteSpace: "normal" }}>
+                      {project_role_description}
+                    </span>
+                  }
+                />
+              </MenuItem>
```

Comment:
I'd like to know how folks feel about including the role description in the menu list item. We've gotten recent user feedback that it would be helpful to provide this kind of context, but this menu gets pretty cramped.

<img width="1421" alt="Screen Shot 2022-10-17 at 11 15 06 AM" src="https://user-images.githubusercontent.com/14793120/196216136-a88890f9-bb8c-4c60-ba91-23986a9ddfc6.png">

Reply:
I initially like the descriptions in there, but dont know if once people get more familiar with the project roles, if it will feel like a pain to have to scroll down to find the role they want. Agree if it were only autocomplete.....

Reply:
@amenity

Reply:
Oops, didn't see this earlier. The position title and description text work as visualized. 

My thought would have been to YOLO widen that drop down menu to the right for more real estate. Sure it would cover up the notes sections temporarily, but the "Notes" title would still be visible and would come back once finished selected.